zerver/views/realm.py

```diff
@@ -148,6 +154,13 @@ def update_realm(
             message_retention_days_raw, Realm.MESSAGE_RETENTION_SPECIAL_VALUES_MAP
         )
 
+    jitsi_server_url: Optional[str] = None
+    if jitsi_server_url_raw is not None:
+        if jitsi_server_url_raw in Realm.JITSI_SERVER_SPECIAL_VALUES_MAP.keys():
+            jitsi_server_url = Realm.JITSI_SERVER_SPECIAL_VALUES_MAP[jitsi_server_url_raw]
+        else:
+            jitsi_server_url = jitsi_server_url_raw
```

Comment:
If you move this to a custom validator function, that could avoid the linter error here.

Reply:
Totally agree with you. Since the function is deemed as an ad-hoc function and we don't have any `lib module` for `jitsi`, should we move the custom validator/parse function to `zerver.lib.utils`?

Reply:
Alright. I've created a custom parse function named `parse_jitsi_server_url` to replace these few lines and the new function is located in `zerver.lib.utils` in the latest commit. However, I am not sure if it is appropriate to place the ad-hoc function in `zerver.lib.utils.py`. Do you have any thoughts on this?